flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/HighAvailabilityServicesUtilsTest.java

```diff
@@ -59,6 +59,22 @@ public void testCreateCustomHAServices() throws Exception {
 		assertSame(haServices, actualHaServices);
 	}
 
+	@Test
+	public void testCreateCustomClientHAServices() throws Exception {
+		Configuration config = new Configuration();
+
+		ClientHighAvailabilityServices clientHAServices = Mockito.mock(ClientHighAvailabilityServices.class);
```

Comment:
I know that this test class already uses `Mockito`. But I would like to avoid adding more Mockito mocks. Instead we could have a very simple `TestingClientHighAvailabilityServices` implementation.

Reply:
Is it reasonable that we remove all Mockito usage in this test case in pr?

Reply:
Done with also remove other `Mockito` usage. I think they are small enough and related to this pr to be included, instead of a dedicated follow up to remove mockito and keep consistency.